rel_ext.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, src_filename_or_triples):
     @staticmethod
     def read_kb_triples(src_filename):
         kb_triples = []
-        with gzip.open(src_filename) as f:
+        with gzip.open(src_filename, encoding='utf8') as f:
```

Comment:
Same as above: we need to add `mode='rt'` to make this work, and that allows us to remove `decode('utf-8')` from the line just below.